bots/image-download

```diff
@@ -224,12 +222,14 @@ def prune_images(force, dryrun):
     # iterate over all visible refs (mostly branches)
     # this hinges on being in the top level directory of the the git checkout
     with remember_cwd():
-        os.chdir(os.path.join(BASE, ".."))
+        os.chdir(os.path.join(BOTS, ".."))
         refs = get_refs()
         # list images present in each branch
         for name, ref in refs.items():
             sys.stderr.write("Considering images from {0} ({1})\n".format(name, ref))
-            for link in get_image_links(ref):
+            for link in get_image_links(ref, "test/images"):
```

Comment:
add a comment here that this is the deprecated, older format?
`# DEPRECATED old branches may still have images in test/images`